app/views/flood_risk_engine/enrollments/steps/_confirmation.html.erb

```diff
@@ -1 +1,35 @@
-confirmation
+<div class="govuk-box-highlight">
+  <h1 class="bold-large"> <%= page_title t('.heading') %></h1>
+  <p>
+    <%= t('.para_text') %>
+    <br />
+    <strong class="heading-medium"><%= form.enrollment.reference_number %></strong>
+  </p>
+</div>
+
+<p class="text"><%= t ".email_sent", email: form.email %></p>
+
+<h2 class="heading-medium">
+  <%= t('.heading_h2') %>
+</h2>
+
+<p class="text"><%= t('.para_text2') %></p>
+<p class="text"><%= t('.para_text3') %></p>
+<p class="text"><%= t('.para_text4') %></p>
+<p class="text"><%= t('.para_text5') %>
```

Comment:
There is a problem with the pattern: `translation_one`, `link`, `translation_two` in that it is difficult to manage across different languages. 

For example if you wanted a link on "you" in "Who are you?", in English the link would go at the start of the string. But if you look at the equivalent Italian, "Tu chi sei", the link would have to be around "Tu" at the start of the string.

Therefore, I think the pattern where you pass the link into the translation as a variable, is a much better pattern. This will allow you to clean up the locale file (see above) and the code here:

```
<p class="text">
<%= t('.para_text5',
    link: link_to('flood risk activities guidance', t('.activities_url'), target: "_blank")
    ).html_safe
%>
</p>
```